var/spack/repos/builtin/packages/py-anuga/package.py

```diff
@@ -13,20 +15,55 @@ class PyAnuga(PythonPackage):
 
     homepage = "https://github.com/GeoscienceAustralia/anuga_core"
     url      = "https://github.com/GeoscienceAustralia/anuga_core/archive/2.1.tar.gz"
+    git      = 'https://github.com/GeoscienceAustralia/anuga_core.git'
 
+    # The git main branch of the repo is now python3-only
+    version('master', branch='main')
     version('2.1', sha256='0e56c4a7d55570d7b2c36fa9b53ee4e7b85f62be0b4c03ad8ab5f51464321d2f')
 
     variant('mpi', default=True, description='Install anuga_parallel')
 
-    # At present AnuGA has only been run and tested using python 2.x.
-    # We recommend python 2.7.
-    depends_on('python@2.6:2.8', type=('build', 'run'))
-    depends_on('py-setuptools@:44', type='build')
-    depends_on('py-numpy@:1.16', type=('build', 'run'))
+    # Non-versioned dependencies for Anuga main and future versions based on python@3.5:
+    depends_on('python@3.5:',            type=('build', 'run'), when='@2.2:')
+    depends_on('gdal+geos+python',       type=('build', 'run'), when='@2.2:')
+    depends_on('py-future',              type=('build', 'run'), when='@2.2:')
+    depends_on('py-matplotlib',          type=('build', 'run'), when='@2.2:')
+    depends_on('py-numpy',               type=('build', 'run'), when='@2.2:')
+    depends_on('py-setuptools',          type=('build'),        when='@2.2:')
+    # Replaces pyar in python3 anuga:
+    depends_on('py-mpi4py',              type=('build', 'run'), when='@2.2:+mpi')
```

Comment:
Unfortunately this needs to be installed whether or not `+mpi` is requested since it appears in `install_requires`. If it isn't installed, setuptools will install it for us

Reply:
ok, done.